node_oracle/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Pull base image
+FROM alpine:latest
+MAINTAINER Scott Ross <sr523@cornell.edu>
+
+# Add Nodejs
+RUN apk add --update nodejs
+
+# Add AWS to install oracle
+RUN apk -Uuv add groff less python py-pip && \
+    pip install awscli && \
+    apk --purge -v del py-pip && \
+    rm /var/cache/apk/*
+
+#INSTALL LIBAIO1 & UNZIP (NEEDED FOR STRONG-ORACLE)
+RUN apk add --update libaio \
+  && apk add --update unzip
+
```

Comment:
best practice would be to collapse this into a single apk instead of three, helps with image bloat